.github/pull_request_template.md

```diff
@@ -1,12 +1,2 @@
 ## Why was this change made?
 
```

Comment:
The question about docs/configs still makes sense.

Reply:
All the documentation is in the repo.  So asking "which" is odd.  Clearly if it's in the diff, then it's been updated. right?

For the configuration question should we be specific and say "shared configs"?  That is the only configuration outside of this repository.

Reply:
From my perspective, the goal is to remind the developer and reviewer to write/review doc and config changes. That is accomplished by the words "documentation" and "configuration". Other than that, not too concerned with the wording.